scripts/e2e/config/kserve/overlay.yaml

```diff
@@ -1,24 +1,28 @@
 apiVersion: apps/v1
 kind: StatefulSet
+metadata:
+  name: kserve-controller-manager
+  namespace: kserve
+$patch: delete
```

Comment:
Not too familiar with kustomize but is this delete operation intended? What are we trying to delete?

Reply:
This is intended. Kserve 0.9.0 allows us to deploy kserve-controller-manager as deployment other than statefulset. However, their released v0.9.0 manifest has both of deployment and stafulset of kserve-controller-manager which needs more editing from users to delete one of them. Hence, by using this kustomize's $patch: delete, I delete statefulset and just use deployment. I have updated PR's description for this.

Reply:
Thanks, Arief!